sdks/python/apache_beam/yaml/yaml_mapping_test.py

```diff
@@ -175,7 +175,7 @@ def test_validate(self):
           label='Errors')
 
   def test_validate_explicit_types(self):
-    with self.assertRaisesRegex(TypeError, r'.*violates schema.*'):
+    with self.assertRaisesRegex(RuntimeError, r'TypeError.*violates schema.*'):
```

Comment:
How about asserting just Exception rather than that it must now be a RuntimeError? (For all of these.)

Reply:
Sure, I can get behind that